### PR TITLE
tests: disable nova from install-snaps test

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Nova ramdomly fails to be installed on gce cloud images.
The error cannot be reproduced on real machines.

### CODE REVIEW

File: tests/nightly/install-snaps/task.yaml [hunk 1]
```diff
@@ -62,7 +62,6 @@ environment:
     SNAP/go: go
     SNAP/juju: juju
     SNAP/neutron: neutron
-    SNAP/nova: nova
```
Comment (bboozzoo) [inline tests/nightly/install-snaps/task.yaml]: Do you have an example error log? Maybe we should pull out nova and neutron into a separate OpenStack oriented test?

Comment (sergiocazzolato) [reply tests/nightly/install-snaps/task.yaml]: @bboozzoo https://travis-ci.org/snapcore/spread-cron/builds/613287688#L2576

Comment (sergiocazzolato) [reply tests/nightly/install-snaps/task.yaml]: @bboozzoo also the gimp snap started failing on ubuntu-16.04-32, I dont know if users are using it on that architecture but seems to be real dependencies issue, what do you think?

Comment (zyga) [reply tests/nightly/install-snaps/task.yaml]: I think those are concerning and should be reported to other teams.

Gimp is a real regression that the desktop team AFAIK knows about now.

Comment (sergiocazzolato) [reply tests/nightly/install-snaps/task.yaml]: @zyga I already reported gimp to the desktop team, they found the problem is for all the architectures but amd64. That should be automatically fixed once the dependency is uploaded again.

**Comment by sergiocazzolato** [comment]
@zyga a time ago I reported that but I remember they couldn't reproduce the error. Could you reproduce it in a standart machine?

**Comment by zyga** [comment]
@sergiocazzolato it's not about me trying to reproduce but about a bug report with the fact that it happened that can collect additional information over time. For amd64 vs other architectures it should be relatively straightforward (snap doesn't run on i386,...). My whole point is not to disable tests but file bugs and, if we ever disable tests, replace them with a bug report URL. The bug report can even have a snapd task to track re-enabling of the relevant test.

**Pull request closed without merging**